Directory.Build.props

```diff
@@ -95,6 +91,10 @@
   <PropertyGroup Condition="Exists('$(SolutionDir)packages\llvm.tools\clang.exe')">
     <ClangExec>"$(SolutionDir)packages\llvm.tools\clang.exe"</ClangExec>
   </PropertyGroup>
+  <!-- If $(VsInstallRoot)\VC\Tools\Llvm\bin\clang.exe exists, set ClangExe to that value -->
+  <PropertyGroup Condition="Exists('$(VsInstallRoot)\VC\Tools\Llvm\bin\clang.exe')">
```

Comment:
@Alan-Jowett seems like this makes CI pick the wrong clang, I don't understand how that works though? IIUC CI is using packages\llvm.tools\clang.exe, so why does that not have the bpf target?

Reply:
NVM, figured out that precedence is last wins over first.